ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/question_answering/SentenceHighlightingQATranslator.java

```diff
@@ -107,89 +102,188 @@ public NDList processInput(TranslatorContext ctx, Input input) {
             NDManager manager = ctx.getNDManager();
             String question = input.getAsString(0);
```

Comment:
let's add static variable for all these magic numbers and add description as well.

Reply:
Updated to:
```
            String question = input.getAsString(MLInput.QUESTION_FIELD);
            String context = input.getAsString(MLInput.CONTEXT_FIELD);
            int chunkNumber = Integer.parseInt(input.getAsString(HIGHLIGHTING_MODEL_CHUNK_NUMBER_KEY));
```